### PR TITLE
Adding storage enclosure keys to confstore

### PR DESCRIPTION
**Pull request by sradhanand-pati**
Signed-off-by: sradhanandpati <sradhanand.pati@seagate.com>

### CODE REVIEW

File: srv/components/system/files/confstore_template.j2 [hunk 1]
```diff
@@ -77,4 +77,21 @@ server_node>{{ machine_id }}>storage_set_id:{{ pillar['cluster'][node]['storage_
 server_node>{{ machine_id }}>cluster_id:{{ cluster_id }}
 server_node>{{ machine_id }}>s3_instances:{{ pillar['cluster'][node]['s3_instances'] }}
 {%- endfor %}
-
+{%- for enclosure in pillar['storage'].keys() %}
+storage_enclosure>{{ pillar['storage'][enclosure]['enclosure_id'] }}>name:{{ enclosure }}
+storage_enclosure>{{ pillar['storage'][enclosure]['enclosure_id'] }}>type:{{ pillar['storage'][enclosure]['type'] }}
```
Comment (andkononykhin2) [inline srv/components/system/files/confstore_template.j2:82]: @sradhanand-pati shouldn't it be 
```
pillar['storage'][enclosure]['enclosure_id']['type']
```
and the same for similar line below.

**Review by andkononykhin2 — CHANGES_REQUESTED** [review]

File: srv/components/system/files/confstore_template.j2 (hunk 1 above)
Comment (83bhp) [reply srv/components/system/files/confstore_template.j2:82]: No, 'type' is under 'enclosure' not under 'enclosure_id'.

**Review by 83bhp — APPROVED** [review]

**Review by mazinamdar — APPROVED** [review]

File: srv/components/system/files/confstore_template.j2 (hunk 1 above)
Comment (ypise) [reply srv/components/system/files/confstore_template.j2:82]: @andkononykhin2 
`pillar['storage'][enclosure]['type']` & `pillar['storage'][enclosure]['enclosure_id']` are at same level in our pillar:
```
storage:
    enclosure-1:
      type: RBOD                      # RBOD/JBOD/Virtual/Other
      controller:
        type:
        primary:
          ip: 10.0.0.2
          port: 80
        secondary:
          ip: 10.0.0.3
          port: 80
        user: manage
        secret:
```

Comment (andkononykhin) [reply srv/components/system/files/confstore_template.j2:82]: Got it. Thank you @83bhp @ypise

**Pull request merged**